### DISCUSSION TITLE
How to override _type attribute

### DISCUSSION THREAD

**Discussion by pierreh** [sensedeep/dynamodb-onetable#504] [General]
According to the documentation:
"OneTable will automatically add a _type attribute to each model that is set to the name of the model. However, you can explicitly define your type attribute in your model schema if you wish."

However when I try to set a custom value like this:
```
_type: {
    type: String,
    value: "t",
},
```

The value is still overriden by OneTable to the name of the model.
How do I do this correctly?

**Comment by mobsense** [comment]
You define the type field via the schema params.

For example:

```
const Schema = {
    format: 'onetable:1.1.0',
    version: 'latest',
    params: {
        isoDates: true,
        nulls: false,
        timestamps: false,
        typeField: '_type',
    },
```


See https://doc.onetable.io/api/table/schemas/params/

**Reply by pierreh** [reply]
@mobsense thanks for the reply!

Is that what the documentation is referring to? Because I did know you can change the field name. But I would also like to be able to change it's default value, by setting the field value under `schema.models`

**Comment by mobsense** [comment]
There can only be one setting for the type field, you cannot vary per model. So it must be defined in the schema params.

**Reply by pierreh** [reply]
Ok, thanks for clarifying.
